questionpy_server/repository/models.py

```diff
@@ -0,0 +1,65 @@
+from dataclasses import dataclass
+from datetime import datetime
+from functools import total_ordering
+
+from pydantic import BaseModel, PositiveInt
+from questionpy_server.utils.manfiest import ComparableManifest, SemVer
+
+
+class RepoMeta(BaseModel):
+    timestamp: datetime
+    size: PositiveInt
+    sha256: str
+    version: int
```

Comment:
Die Version sollte etwas eindeutiger benannt werden und an erster Stelle stehen. Es könnte ansonsten auch als eine fortlaufende Nummerierung verstanden werden, um Änderungen im Repo zu kennzeichnen.

Vielleicht `qpy_repository_schema_version`?

Bitte schreibe noch bei der Gelegenheit in dieser Datei auch mehr Kommentare und beschreibe die einzelnen Felder.